packages/terra-button/src/Button.jsx

```diff
@@ -235,6 +241,9 @@ class Button extends React.Component {
         onClick={onClick}
         onFocus={onFocus}
         href={href}
+        ref={(inputRef) => {
+          if (refCallback) refCallback(inputRef);
```

Comment:
This can be shortened to `ref={refCallback}`

Reply:
Good catch, updated